ipa-core/src/helpers/transport/stream/axum_body.rs

```diff
@@ -25,6 +24,10 @@ impl WrappedAxumBodyStream {
     pub fn empty() -> Self {
         Self::new(Body::empty())
     }
+
+    pub fn from_bytes_stream<S: BytesStream + 'static>(stream: S) -> Self {
```

Comment:
`WrappedAxumBodyStream` implements `BytesStream` as well, so this will lead to conflicting implementations of `From` because there always exists `From<T> for T`